MANIFEST.in

```diff
@@ -1 +1,3 @@
-include LICENSE
\ No newline at end of file
+include LICENSE
+
+graft websockets/py35
```

Comment:
If I understand correctly, this has the effect of including the `websockets/py35` package even if it isn't in `packages` -- that is, when packaging on Python < 3.5. Can you confirm?